### ISSUE TITLE
Adserver

### ISSUE THREAD

**Issue by IsraeliUser** [easylist/easylist#7749]
`||sekindo.com^$third-party`

https://github.com/easylist/easylist/commit/3a101a8cdb5b5d5ccad4aa6364a903c8579b01d1#comments

This adserver serves ads in many Israeli websites.
You can use whitelist/exceptions to fix false positives.

**Comment by ryanbr** [comment]
example of ads? https://appleinsider.com/articles/21/04/24/aclu-eff-ask-supreme-court-to-review-border-device-search-policies  seems okay

**Comment by llacb47** [comment]
Sekindo is the previous name of Primis. See how they describe themselves:
![image](https://user-images.githubusercontent.com/16567977/116489269-2384fe80-a884-11eb-8aa7-32a0df345ab7.png)

With sekindo.com totally unblocked you get this ugly video widget that is not related to the content on pages like:
`https://awfulannouncing.com/podcasts/rex-chapman-launching-podcast-interviewing-athletes-about-troubles-with-law.html`
`https://thecomeback.com/golf/european-tour-player-sets-world-record-for-farthest-golf-shot-caught-in-a-moving-car.html`.
`https://themeparkreview.com/`
`https://www.onlineloancalculator.org/`

However, the widget is totally nonfunctional due to this rule in Easylist.
![image](https://user-images.githubusercontent.com/16567977/116489697-19173480-a885-11eb-8bfc-d4c4f39f7cf9.png)

I wonder why something that's not meant for advertising would rely on this request... 🤔🤔🤔

So in conclusion, you removing it from easylist changed nothing --  except now there's an unneeded third-party connection on many sites to sekindo on many sites.
